.github/workflows/ci.yml

```diff
@@ -239,10 +239,16 @@ jobs:
           sed -i '3i   set(ENV{CMAKE_POLICY_VERSION_MINIMUM} 3.5)'  ${env:VCPKG_INSTALLATION_ROOT}/scripts/ports.cmake
           sed -i '4i endif()'                                       ${env:VCPKG_INSTALLATION_ROOT}/scripts/ports.cmake
           vcpkg install --triplet x64-windows pkgconf libevent
+
+      - run: rm 'C:/Program Files/OpenSSL/' -rf
+        shell: bash
+
+      - run: choco install openssl
```

Comment:
You can remove this added section (including the `rm 'C:/...`) and just add openssl to line 236, 
i.e
`choco install -y memurai-developer openssl`

Reply:
there is a openssl 1.1.1w at 'C:/Program Files/OpenSSL/`

Reply:
Ah, I see. Is the there any problem with default version already installed? Maybe you want to make sure a maintained version of openssl works?

Reply:
We don't even enabled TLS on windows ci before this, so I think it's fine to just add latest openssl version?